src/testing/loadtest-azure/scripts/catalogue-test.jmx

```diff
@@ -0,0 +1,231 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can we change that filename to american english? :)

Reply:
renamed